src/shared/prompts/en/parts/workflow_wide_rule.md

```
@@ -0,0 +1,6 @@
+<!--
+  template: parts/workflow_wide_rule
+  vars: ref, content
+-->
+### Workflow-wide rule: {{ref}}
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)